packages/core/src/media/controller.ts

```diff
@@ -49,6 +49,8 @@ export type MediaControllerState<TElement = void> = {
   priority: boolean;
   /** The preload option passed in to Player */
   preload: 'full' | 'metadata' | 'none';
+  /** The walletId for the viewer passed in to Player */
+  walletId: string;
```

Comment:
@victorges what do you think about this naming?

Reply:
Hmm, I think I'd prefer the more generic "viewerId", as not necessarily these are wallets (maybe just an email for example). I also liked that idea from Hunter to have these be an object with a `type` field so we can represent different kinds of wallets and IDs. e.g.
```suggestion
  viewerId: { type: "unverified", value: string } | { type: "evm", address: `0x${string}`, proof: string }
```

Or something like that. WDYT?